web/modules/custom/mymodule/src/Plugin/Field/FieldWidget/RealNameDefaultWidget.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\myplugin\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\WidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implementation of RealNameDefaultWidget.
+ *
+ * @FieldWidget(
+ *   id = "RealNameDefaultWidget",
+ *   label = @Translation("Real Name"),
+ *   field_types = {
+ *     "real_name"
+ *   }
+ * )
+ */
+class RealNameDefaultWidget extends WidgetBase {
+
+  /**
+   * Define the form for the field type.
+   */
+  public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
+    $element['first_name'] = [
+      '#type' => 'textfield',
+      '#title' => 'First Name',
```

Comment:
@anitha-innoraft - Title should be translatable. Please use `$this->t()`

Reply:
Done